### ISSUE TITLE
Buffer Encoding Error in node.js v6.0.0

### ISSUE THREAD

**Issue by DustinVenegas** [Azure/azure-xplat-cli#2829]
CLI Version: **0.9.20**
OS Type: Mac
Installation via:  npm

Mode: **ARM**

Description:
All `azure` commands output an error after updating to node.js 6.0.0. This is likely related to [this pull request](https://github.com/nodejs/node/pull/4514) in node.js. The node-postgres project encountered a similar error in [this issue](https://github.com/brianc/node-postgres/issues/1000). 

Steps to reproduce:
1) Install node.js v6.0.0
2) Then run any `azure` command

Error stack trace:

```
buffer.js:106
      throw new Error(
      ^

Error: If encoding is specified then the first argument must be a string
    at new Buffer (buffer.js:106:13)
    at Interactor.__.extend.initProgressBars (/usr/local/lib/node_modules/azure-cli/lib/util/interaction.js:41:24)
    at new Interactor (/usr/local/lib/node_modules/azure-cli/lib/util/interaction.js:32:8)
    at new AzureCli (/usr/local/lib/node_modules/azure-cli/lib/cli.js:43:22)
    at /usr/local/lib/node_modules/azure-cli/bin/azure.js:35:11
    at Object.exports.isTelemetryEnabled (/usr/local/lib/node_modules/azure-cli/lib/util/utilsCore.js:181:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/azure-cli/bin/azure.js:27:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
```

**Comment by amarzavery** [comment]
@DustinVenegas 
Fixed in the PR #2821 that is merged in the dev branch. The new version will be available ~5/4.
Duplicate - #2825 
If you need the bits from dev branch follow the instructions [here](https://github.com/Azure/azure-xplat-cli#installation-from-a-particular-branch-of-this-repository).

I will keep this open for now.

**Comment by DustinVenegas** [comment]
Thank you @amarzavery! I missed the duplicate in my search. I'll start watching #2825.
